Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM ubuntu
```

Comment:
Could you add a comment on top of this file with a few words about what it's for. I can see myself getting very confused in a few months about seeing a Dockerfile in the root of the project where I can't tell what it does by its contents.

Questions to address in the comment: who and when and why would build this image and how would they run it (volume mounts, ports etc)?

Reply:
I guess you can just point the reader to the devcontainer config file

Reply:
@iliakur 
I added a comment in the Dockerfile and moved it to the `.devcontainer` since it is not mandatory to have it in the root folder.

Reply:
Ah, the combination of those two actions clarifies things very well, thanks!